std/random.d

```diff
@@ -1820,6 +1867,13 @@ $(D partialShuffle) was called.
 
 $(D r) must be a random-access range with length.  $(D n) must be less than
 or equal to $(D r.length).  If no RNG is specified, $(D rndGen) will be used.
+
+Params:
+    r = random-access range whose elements are to be shuffled
+    n = number of elements of $(D r) to shuffle (counting from the beginning);
+        must be less than $(D r.length)
+    gen = (optional) random number generator to use; if not
```

Comment:
And, reply ditto. :-)  (Small aside: the reason this is done with two separate function overloads, one with and one without the `gen` parameter, is because of https://issues.dlang.org/show_bug.cgi?id=2803.)